# identifies the robot
string instance_name
---
# a string in json format which represents the configuration data for the robot
# {
#     "ROBOT_NAME": {
#         "topics": [
#             {
#                 "name": "TOPIC NAME",
#                 "msg": "TOPIC MESSAGE TYPE",
#                 "type": "COMMUNICATION TYPE"
#             }
#         ]
#     }
# }
string json_format
